packages/react-core/src/components/Chip/__tests__/ChipGroup.test.tsx

```diff
@@ -6,65 +6,63 @@ import userEvent from '@testing-library/user-event';
-    const { asFragment } = render(
-      <ChipGroup>
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
+test('chip group default', () => {
+  render(
+    <ChipGroup>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
 
-    expect(asFragment()).toMatchSnapshot();
-  });
+  expect(screen.getByRole('group')).toHaveClass('pf-v5-c-chip-group');
+});
 
-  test('chip group with category', () => {
-    const { asFragment } = render(
-      <ChipGroup categoryName="category">
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('chip group with category', () => {
+  render(
+    <ChipGroup categoryName="category">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByText('category')).toBeVisible();
+});
 
-  test('chip group with closable category', () => {
-    const { asFragment } = render(
-      <ChipGroup categoryName="category" isClosable>
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('chip group with closable category', () => {
+  render(
+    <ChipGroup categoryName="category" isClosable>
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByLabelText('Close chip group').closest('div')).toHaveClass('pf-v5-c-chip-group__close');
+});
 
-  test('chip group expanded', async () => {
-    const user = userEvent.setup();
+test('chip group expanded', async () => {
+  const user = userEvent.setup();
 
-    render(
-      <ChipGroup>
-        <Chip>1</Chip>
-        <Chip>2</Chip>
-        <Chip>3</Chip>
-        <Chip>4</Chip>
-      </ChipGroup>
-    );
+  render(
+    <ChipGroup>
+      <Chip>1</Chip>
+      <Chip>2</Chip>
+      <Chip>3</Chip>
+      <Chip>4</Chip>
+    </ChipGroup>
+  );
 
-    const moreText = screen.getByText('1 more');
-    expect(moreText).toBeInTheDocument();
+  const moreText = screen.getByText('1 more');
+  expect(moreText).toBeInTheDocument();
 
-    await user.click(moreText);
-    expect(screen.getByText('Show Less')).toBeInTheDocument();
-  });
+  await user.click(moreText);
+  expect(screen.getByText('Show Less')).toBeInTheDocument();
+});
 
-  test('chip group will not render if no children passed', () => {
-    render(<ChipGroup />);
-    expect(screen.queryByRole('group')).toBeNull();
-  });
+test('chip group will not render if no children passed', () => {
+  render(<ChipGroup />);
+  expect(screen.queryByRole('group')).toBeNull();
+});
 
-  test('chip group with category and tooltip', () => {
-    const { asFragment } = render(
-      <ChipGroup categoryName="A very long category name">
-        <Chip>1.1</Chip>
-      </ChipGroup>
-    );
-    expect(asFragment()).toMatchSnapshot();
-  });
+test('chip group with category and tooltip', () => {
+  render(
+    <ChipGroup categoryName="A very long category name">
+      <Chip>1.1</Chip>
+    </ChipGroup>
+  );
+  expect(screen.getByText('A very long category name')).toBeVisible();
```

Comment:
Good point @thatblindgeye, I do agree this would be better suited as an integration test. The method the Truncate test uses to mock the tooltip doesn't seem to work here.